pkg/printers/internalversion/printers.go

```diff
@@ -1167,12 +1164,10 @@ func printNode(obj *api.Node, options printers.PrintOptions) ([]metav1alpha1.Tab
 	if obj.Spec.Unschedulable {
 		status = append(status, "SchedulingDisabled")
 	}
-	role := findNodeRole(obj)
-	if role != "" {
-		status = append(status, role)
-	}
 
-	row.Cells = append(row.Cells, obj.Name, strings.Join(status, ","), translateTimestamp(obj.CreationTimestamp), obj.Status.NodeInfo.KubeletVersion)
+	roles := findNodeRoles(obj)
+
+	row.Cells = append(row.Cells, obj.Name, strings.Join(status, ","), strings.Join(roles, ","), translateTimestamp(obj.CreationTimestamp), obj.Status.NodeInfo.KubeletVersion)
```

Comment:
Should we still print this column if `len(roles) == 0`? I have no specific opinion on that...
Or should we default to just print `node`?
(just throwing out ideas here)